docs/submission/dictionary-overview.md

```diff
@@ -52,4 +52,252 @@ The set of core clinical fields were defined by the [Tissue & Clinical Annotatio
+| Stage 1 | All of the following: <ul><li>Hemoglobin value > 10 g/dl</li><li>Serum calcium value normal or <10.5 mg/dL</li><li>Bone X-ray, normal bone structure (scale 0), or solitary bone plasmactyoma only</li><li>Low M-component production rates IgG value < 5g/dL; IgA value < 3 g/dL</li><li>Urine light chain M-component on electrophoresis < 4 g/24h</li></ul> |
+| Stage II | Fitting neither Stage I nor Stage III |
+| Stage III | One or more of the following: <ul><li>Hemoglobin value < 8.5 g/dL</li><li>Serum calcium value > 12 mg/dL</li><li>Advanced lytic bone lesions (scale 3)</li><li>High M-component production rates IgG value > 7 g/dL; IgA value > 5 g/dL</li><li>Urine light chain M-component > 12 g/24h</li></ul> |
+Reference: [https://www.myeloma.org/durie-salmon-staging](https://www.myeloma.org/durie-salmon-staging)
+
+
+### FIGO staging of Cervical carcinomas
+
+| Stage | Description |
+|-------|-------------|
+| Stage IA | Invasive cancer identified only microscopically. Invasion is limited to measured stromal invasion with a maximum depth of 5 mm and no wider than 7 mm. |
+| Stage IA1 | Measured invasion of the stroma no greater than 3 mm in depth and no wider than 7 mm diameter.|
+| Stage IA2 | Measured invasion of stroma greater than 3 mm but no greater than 5 mm in depth and no wider than 7 mm in diameter.|
+| Stage IB | Clinical lesions confined to the cervix or preclinical lesions greater than Stage IA. All gross lesions even with superficial invasion are Stage IB cancers.|
+| Stage IB1 | Clinical lesions no greater than 4 cm in size.|
+| Stage IB2 | Clinical lesions greater than 4 cm in size.|
+| Stage IIA | No obvious parametrial involvement. Involvement of up to the upper two-thirds of the vagina.|
+| Stage IAB | Obvious parametrial involvement, but not into the pelvic sidewall.|
+| Stage IIIA | No extension into the pelvic sidewall but involvement of the lower third of the vagina.|
+| Stage IIIB | Extension into the pelvic sidewall or hydronephrosis or non-functioning kidney.|
+| Stage IVA | Spread of the tumour into adjacent pelvic organs.|
+| Stage IVB | Spread to distant organs.|
+Reference: [https://screening.iarc.fr/viaviliappendix1.php](https://screening.iarc.fr/viaviliappendix1.php)
+
+### Binet staging system
+
+| Stage | Description |
+|-------|-------------|
+| Stage A | Fewer than 3 groups of enlarged lymph nodes (lymphadenopathy) and a high white blood cell count |
+| Stage B | More than 3 groups of enlarged lymph nodes and a high white blood cell count 
+| Stage C | Enlarged lymph nodes or spleen, a high white blood cell count, and low red blood cell or platelet counts |
+Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition (p. 975). American College of Surgeons.
+
+
+## Tumour grading classifications
+
+
+### Two-tier grading system
+
+| Grade |
+|-------|
+| Low grade |
+| High grade | 
+Reference: Reference: Amin, Mahul B.. AJCC Cancer Staging Manual, Eighth Edition
+
+
+### Three-grade system
```

Comment:
should this be "Three-tier Grading System"?

Reply:
Yes it should. I corrected this.